### ISSUE TITLE
Default memory size

### ISSUE THREAD

**Issue by GlaceAiels** [mupen64plus-ae/mupen64plus-ae#605]
Is there a way to change my default memory size from the default to 8mb? I am trying to play super Mario star road and I cannot because of the memory. If there's a way please let me know or put it in the next update please.

**Comment by fzurita** [comment]
Yes, create a new emulation profile. In there you can set your expansion pak size.

Go to profiles, then emulation profiles. In there, you can use the top menu to create s new one.

Also, I'm not sure if start road is compatible with Mupen64plus, but feel free to try it.
